.github/workflows/push-rdev.yml

```diff
@@ -24,14 +24,24 @@ permissions:
 
 jobs:
   build-push-images:
+    strategy:
+      matrix:
+        image:
+          - frontend # pushed both the frontend and backend images
```

Comment:
not doubting you but how is this also building the backend? Where's the dependency pick-up happening?

Reply:
by putting them in the same [profile](https://github.com/chanzuckerberg/single-cell-data-portal/blob/827abee7af280e8325322e2f11e0f62ef7d05a7e/.happy/config.json#L18-L22) for [frontend](https://github.com/chanzuckerberg/single-cell-data-portal/blob/827abee7af280e8325322e2f11e0f62ef7d05a7e/docker-compose.yml#L42) and [backend](https://github.com/chanzuckerberg/single-cell-data-portal/blob/827abee7af280e8325322e2f11e0f62ef7d05a7e/docker-compose.yml#L295C13-L295C13)